integration-libs/cpq-quote/cpq-quote-discount/cpq-qute.service.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// cpq-quote-status.service.ts
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class CpqQuoteService {
+  private isFlagSubject = new BehaviorSubject<boolean>(true); // Default value is true
+  isFlag$ = this.isFlagSubject.asObservable();
+
+  setIsFlag(value: boolean) {
```

Comment:
@Jali-Khusi, your getter is called `getFlag()` but your setter is called `setIsFlag()`.
Could you rename to `setFlag()`?